clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -407,7 +409,7 @@ public static Object[][] sortedInputAndAAConfigProvider() {
       createReplicationMetadataWithValueSchemaId(DELETE_KEY_FOO_TIMESTAMP, DELETE_KEY_FOO_OFFSET, EXISTING_SCHEMA_ID);
 
   private boolean databaseChecksumVerificationEnabled = false;
-  private KafkaConsumerServiceStats kafkaConsumerServiceStats = mock(KafkaConsumerServiceStats.class);
+  private AggKafkaConsumerServiceStats _kafkaConsumerServiceStats = mock(AggKafkaConsumerServiceStats.class);
```

Comment:
We don't use underscore in for variable in Venice repo style. Could you remove that?